src/raytracer2018.nim

```diff
@@ -1401,6 +1401,60 @@ proc computeMagnetTransmission(
     #echo "axion mass in [eV] ", mAxion, " effective photon mass in [eV] ", effPhotonMass
     result = cos(ya) * probConversionMagnetGas * absorbtionXrays # for setup with gas
 
+proc lineIntersectsOpaqueTelescopeStructures(
+  expSetup: ExperimentSetup,
+  testVector, vectorEntranceXRTCircular, pointExitCB, pointEntranceXRT: Vec3[float],
+  onlyInnerPart = false
+     ): bool =
+
+  ## XXX: `onlyInnerPart` is a hack? for the case where we only want to enter the
+  ## first branch of the `if vectorEntrance...` for `esBabyIAXO`.
+  ##
+  ## XXX: Also the expSetup should become a `telescopeKind` case, no?
+  ##
+  ## XXX: can we please clean up the esBabyIAXO branch? Remove the commented code?
```

Comment:
Yes, most of the comments are not needed anymore (except for the strip one and the first descriptive one I guess